Fix FAQ for UDP is supported (#5178)
istio/istio.io: content/en/faq/traffic-management/naming-port-convention.md

weight: 50
---

> Currently, Istio supports TCP based protocols. In additional to this, Istio provides additional functionality, such as routing and metrics, from other protocols such as `http` and `mysql`.

Reviewer: ```suggestion
Currently, Istio supports TCP based protocols. Additionally, Istio provides functionality such as routing and metrics for other protocols such as `http` and `mysql`.
```
